src/syntax/instrumentPpx.ml

```diff
@@ -208,6 +208,9 @@ let translate_pattern =
        it makes no sense to match a second time for effects such as
        exceptions. *)
     | Ppat_exception _ -> []
+    | Ppat_open (c, p') ->
+      translate mark p'
+      |> List.map (fun (marks, p'') -> marks, Pat.open_ ~loc ~attrs c p'')
```

Comment:
The code change LGTM, but I would suggest that it be moved above the `Ppat_exception` case. Otherwise, it looks like the comment above applies to `Ppat_open` as well.